### ISSUE TITLE
Cannot read properties of undefined (reading '_emitter') when using HTTP WebHooks 

### ISSUE THREAD

**Issue by ralph44** [crisp-im/node-crisp-api#51]
# Description

When using the Crisp library with the RTM_MODES.WebHooks mode, the following error is thrown:

`TypeError: Cannot read properties of undefined (reading '_emitter') at \node_modules\crisp-api\lib\crisp.js:834:31`

This error occurs in the _connectLoopback method of the crisp-api library, which is called by the _prepareBroker method. The problem appears to be that the this reference is not properly bound to the Crisp instance, which causes the _emitter property to be undefined.

## Steps to Reproduce:

1. Initialize a Crisp instance with the RTM_MODES.WebHooks mode.
2. Call the on method to bind an event listener.
3. Wait for the error to occur.

### Expected Result:

No errors should occur, and the event listener should be properly bound to the Crisp instance.

### Actual Result:

The error cannot read properties of undefined (reading '_emitter') is thrown, and the event listener is not properly bound to the Crisp instance.

## Potential Solution:

It appears that adding the following line of code `var self = this;` to the _connectLoopback method in the crisp-api library could fix the problem:

```
_connectLoopback : function() {
    var self = this;

    return Promise.resolve()
      .then(function() {
        // Assign emitter to loopback
        self._loopback = self._emitter;

        // Unstack broker bind hooks immediately
        self._unstackBrokerBindHooks(self._loopback);

        return Promise.resolve();
      });
  },
```

This would create a local variable self that references the current Crisp instance and can be used to access its properties and methods.  

Version Information:

crisp-api version: 7.4.1
Node.js version: v16.19.0
Operating System: Windows 10

Additional Information:

**I'm not entirely certain that this is a proper solution, and there may be other underlying issues causing the error. Maybe it is the way i implemented the crisp-api into my service. It's also possible that adding the var self = this; line may have unintended side effects. But other functions are using the same line. **

I tried to followed these [instructions](https://docs.crisp.chat/guides/web-hooks/plugin-hooks/#receiving-web-hooks-using-a-library). I would loved to just use this [example](https://github.com/crisp-im/node-crisp-api/blob/master/examples/events_webhooks.js), but i did not want to mix the express code with raw http server code.

**Comment by valeriansaliou** [comment]
Hello! Could you provide the full code leading to the error?

**Comment by valeriansaliou** [comment]
Fixed in `v8.1.0`, related to #58
